src/content/topics/home/account-security/sso/index.mdx

```diff
@@ -108,6 +108,15 @@ You can configure your identity provider to send the following attributes when t
   </CalloutDescription>
 </Callout> 
 
+
+## Required attributes
```

Comment:
NameID isn't an attribute (that has a specific meaning in SAML), it's a field. It's required just as part of the SAML protocol.
Maybe this could be "NameID Format"?